Makefile

```diff
@@ -1,13 +1,28 @@
 .SUFFIXES: .pdf .tex .bib $(SUFFIXES)
 
-BUILD=build
-TARGETS=acs-category-theory-notes.pdf
+FLAGS := -quiet
+BUILD := build
+SRC :=  lecture1.tex \
+	lecture2.tex \
+	lecture3.tex \
+	lecture4.tex \
+	lecture5.tex \
+	lecture6.tex \
+	lecture7.tex \
+	lecture8.tex \
+	lecture9.tex
```

Comment:
This is fine, but I would much prefer it if file splitting was in a separate pull request to the content from lecture 9.

Reply:
Yeah fair enough, that's on me, it's just lecture 9 had already started, the compile times were horrendous and I didn't want to perform git sorcery on a Sunday morning.

Reply:
No worries. How long does it take you to compile? It hardly takes me a second...